eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/parameters/InputParameters.java

```diff
@@ -86,6 +99,7 @@ public JobParameters toJobParameters() {
         return new JobParametersBuilder()
                 .addString("assemblyAccession", assemblyAccession)
                 .addString("projectAccession", projectAccession)
+                .addString("projects", CollectionUtils.isEmpty(projects) ? "" : String.join(",", projects))
```

Comment:
Apparently, parameters "string_val" in the Spring job parameters table has a [character limit of 250](https://docs.spring.io/spring-batch/docs/current/reference/html/schema-appendix.html). Not sure if we will exceed it, but may be we can put in a check somewhere to check that upfront.

Reply:
Also suggest trimming whitespace before and after the comma for each study...